pyqtgraph/flowchart/Node.py

```diff
@@ -462,6 +462,12 @@ def keyPressEvent(self, ev):
                 self.on_update()
                 return
         super().keyPressEvent(ev)
+        
+    def mousePressEvent(self, ev):
+        if ev.button() == QtCore.Qt.MouseButton.LeftButton:
+            self.setTextInteractionFlags(QtCore.Qt.TextInteractionFlag.TextEditorInteraction)
+        elif ev.button() == QtCore.Qt.MouseButton.RightButton:
+            self.setTextInteractionFlags(QtCore.Qt.NoTextInteraction)
```

Comment:
Slight change request, I think this should be 

```python
self.setTextInteractionFlags(QtCore.Qt.TextInteractionFlag.NoTextInteraction)
```

PyQt6 mandates that the full enum namespace is used.  On PySide2/6, and PyQt5, the abbreviated namespace works.  If you would make the change that would be great.

Reply:
Done.

Reply:
Also added the code to set **focus on text** https://github.com/pyqtgraph/pyqtgraph/pull/2236/commits/b0880ea1b723e9cb7064e3d36d07b4f0b929c987